phpcs.xml.dist

```diff
@@ -65,4 +64,10 @@
     <rule ref="SlevomatCodingStandard.Operators.DisallowEqualOperators.DisallowedNotEqualOperator">
         <exclude-pattern>lib/Doctrine/DBAL/Schema/Comparator.php</exclude-pattern>
     </rule>
+
+    <!-- db2_autocommit() expects bool, but DB2_AUTOCOMMIT_ON/DB2_AUTOCOMMIT_OFF are of int type
```

Comment:
It's the opposite: `DB2_AUTOCOMMIT_ON` and `DB2_AUTOCOMMIT_OFF` are of `int` type but `db2_autocommit()` expects `bool` (https://github.com/php/pecl-database-ibm_db2/pull/8#discussion_r258274427).

Reply:
I think that's exactly what's written in the comment. 🤔

Reply:
From the original comment, it follows that the constants are defined incorrectly because they are after "but", however, the bug is in the method definition. Or maybe it's just me? Feel free to leave as is if you think it's good enough.

Reply:
Reworded. 👍